src/components/Application/ProgressBar.tsx

```diff
@@ -32,6 +39,11 @@ const ProgressBar: React.FC<ProgressBarProps> = ({
     return status ? indicator[status] : null
   }
 
+  const getPreviousPageIsValid = (sectionCode: string, pageNumber: number) => {
```

Comment:
I kind of like functions that return booleans to start with `is` -- so `isPreviousPageValid`. 

No biggie though, I just think it reads clearer (as opposed to `get` which suggests I'm expecting a spefiic value back).

Reply:
Oh, I agree. That's a good idea.
Will change this now and start merging PRs!